src/gameobjects/UpdateList.js

```diff
@@ -18,6 +18,7 @@ var SceneEvents = require('../scene/events');
  * Some or all of these Game Objects may also be part of the Scene's [Display List]{@link Phaser.GameObjects.DisplayList}, for Rendering.
  *
  * @class UpdateList
+ * @extends Phaser.Structs.ProcessQueue.<Phaser.GameObjects.GameObject>
```

Comment:
Please try generating the defs after doing this, to make sure it doesn't break them. You can use the command `npm run tsdev` to check and view the contents of the output.txt it builds.

Reply:
yes I did this and tested, so this is the reason the commit has `phaser.d.ts` changes in it. remove the `phaser.d.ts` from the commit is ok, `cause users can re-gnerate it by npm run command still.